chsdi/templates/htmlpopup/phosphor.mako

```diff
@@ -0,0 +1,5 @@
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c,lang)">
+   <tr><td width='200'>${_('ch.bafu.gewaesserschutz-diffuse_eintraege_phosphor.phosphor')}</td><td>${c['properties']['phosphor'] or '0'}</td></tr>
```

Comment:
If there is a `None` value you'll get a `0`.
Can you have such a use case?
I am also wondering about the number of significant digits we should display in a popup. For instance I get `0.460325688124`. Is that ok?